Index: target/m2e-wtp/web-resources/META-INF/MANIFEST.MF
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Manifest-Version: 1.0\r\nBuilt-By: yuanz\r\nBuild-Jdk: 11.0.21\r\nCreated-By: Maven Integration for Eclipse\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/target/m2e-wtp/web-resources/META-INF/MANIFEST.MF b/target/m2e-wtp/web-resources/META-INF/MANIFEST.MF
--- a/target/m2e-wtp/web-resources/META-INF/MANIFEST.MF	(revision fb49c34cb9b2173f7ba34649f50c6d5b45c0856e)
+++ b/target/m2e-wtp/web-resources/META-INF/MANIFEST.MF	(date 1711943555650)
@@ -1,5 +1,5 @@
 Manifest-Version: 1.0
-Built-By: yuanz
+Built-By: goott4
 Build-Jdk: 11.0.21
 Created-By: Maven Integration for Eclipse
 
